File: server/src/readjson/PotionRadius.java
package readjson;

import java.io.Serializable;

public class PotionRadius implements Serializable {
    private double value;
    private double increasePerRank;

    public double getRadius(int rank) {
        return this.value + this.increasePerRank * (rank - 1);
    }
}

File: client/src/modules/readjson/PotionRadius.js
/**
 * Created by Team 2 - LongLH - GDF 17 on 12/12/2022.
 */


var PotionRadius = cc.Class.extend({

    value: Number.prototype,
    increasePerRank: Number.prototype,

    getRadius: function (rank) {
        return this.value + this.increasePerRank * (rank - 1);
    }

});